src/components/ReleaseService/ReleasePlan/releaseplan-actions.tsx

```diff
@@ -11,6 +11,13 @@ export const useReleasePlanActions = (obj: ReleasePlanKind) => {
   const [canDelete] = useAccessReviewForModel(ReleasePlanModel, 'delete');
   const [canUpdate] = useAccessReviewForModel(ReleasePlanModel, 'update');
   return [
+    {
+      label: 'Trigger release plan',
+      id: `trigger-releaseplan-${obj.metadata.name}`,
+      cta: {
+        href: `/application-pipeline/release/workspaces/${workspace}/release-plan/trigger`,
+      },
```

Comment:
Can we add access check review to see if the user has access to `create` a release, if not disable the action with tooltip ?